functions/go/format/README.md

```diff
@@ -4,16 +4,16 @@
 
 <!--mdtogo:Short-->
 
-Format the field ordering in resources.
+Sort resource fields, fix indentation and quote ambiguous string values.
```

Comment:
`Format the resources` is not helpful :) What you have there is better than what I suggested, so ignore my suggestion.

Reply:
Yes, the point is not being tautological. This can combine both suggestions:

`Format resources by sorting fields, fixing indentation, etc.`.

Although I think this doesn't actually capture all formatting changes. Does it?

Reply:
Updated.